### ISSUE TITLE
a11y: Color Contrast - Branding

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13709]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF
Ratio: 3.16:1 
Main Content
- Private Label input field (on hover and focus)
- Upload Light Logo, Upload Dark Logo, Upload Light Banner, Upload Dark Banner, Upload Favicon, Link Example (available after checking Use a Custom Logo, Use a Custom Banner, Use a Custom Background, Use a Custom Favicon, Use a Custom Link Color checkboxes) 

Foreground Text Color: #141419
Background Text Color: #297DB4
Ratio: 4.09:1 
Main Content
- Upload Light Logo, Upload Dark Logo, Upload Light Banner, Upload Dark Banner, Upload Favicon (on hover ) 
(available after checking Use a Custom Logo, Use a Custom Banner, Use a Custom Background, Use a Custom Favicon) 

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content 
- Upload Light Logo, Upload Dark Logo, Upload Light Banner, Upload Dark Banner, Upload Favicon (on focus) (available after checking Use a Custom Logo, Use a Custom Banner, Use a Custom Background, Use a Custom Favicon)
- Apply Button 

Foreground Text Color: #FFFFFF
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Apply button (on focus) 

Foreground Text Color: #B6B6C2
Background Text Color: #FFFFFF
Ratio: 2.00:1 
Main Content 
- #3d98d3 ( available after checking Use a Custom Color and Use a Custom Link Color checkboxes)  

 

         "

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
>
> Instances:
> Foreground Text Color: #2F68DF
> Background Text Color: #2F68DF
> Ratio: 1:1
> Main Content
> - Upload Light Logo, Upload Dark Logo, Upload Light Banner, Upload Dark Banner, Upload Favicon 
> (available after checking Use a Custom Logo, Use a Custom Banner, Use a Custom Background, Use > a Custom Favicon) (default state)
> 
> Foreground Text Color: #2F68DF
> Background Text Color: #1D51BE
> Ratio: 1.4:1
> Main Content 
> - Upload Light Logo, Upload Dark Logo, Upload Light Banner, Upload Dark Banner, Upload Favicon > (on focus and hover state) (available after checking Use a Custom Logo, Use a Custom Banner, Use a Custom Background, Use a Custom Favicon)
>
>
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 239
